crates/abq_utils/src/net_protocol/runners.rs

```diff
@@ -343,6 +365,112 @@ impl Manifest {
     }
 }
 
+#[cfg(test)]
+mod test_manifest {
+    use crate::net_protocol::{
+        queue::TestSpec,
+        runners::v0_2::{Group, Manifest, Test, TestOrGroup},
+        workers::GroupId,
+    };
+    use serde_json::Map;
+
+    fn flattened(members: Vec<TestOrGroup>) -> Vec<TestSpec> {
+        Manifest {
+            members,
+            init_meta: Map::new(),
+        }
+        .flatten()
+        .0
+    }
+
+    fn test(name: &'static str) -> TestOrGroup {
+        TestOrGroup::Test(Test {
+            id: name.to_string(),
+            tags: vec![],
+            meta: Map::new(),
+        })
+    }
+
+    fn group(members: Vec<TestOrGroup>) -> TestOrGroup {
+        TestOrGroup::Group(Group {
+            name: String::new(),
+            members,
```

Comment:
fwiw you can change this to
```suggestion
    fn group(members: impl Into<Vec<TestOrGroup>>) -> TestOrGroup {
        TestOrGroup::Group(Group {
            name: String::new(),
            members: members.into(),
```

then you can avoid a bunch of `vec![]`s in the tests, and just use `[]`